### DISCUSSION TITLE
loadPixels

### DISCUSSION THREAD

**Discussion by hello-Ryan** [P5-wrapper/react#287] [Q&A] [answered]
Hello,
I'm trying to create some fractals in a nextjs react project. I'm not able to use p5.loadPixels() to get the pixel array as im getting the error TypeError: Cannot read properties of undefined (reading 'loadPixels').

Is this function supported right now?

Any ways to fix?

**Comment by jamesrweb** [comment]
@hello-Ryan Can you share your code?

Also, why is the conversation marked as closed? I didn't see it until now. Did you resolve it yourself, or what? 😅

**Reply by hello-Ryan** [reply, marked as answer]
> @hello-Ryan Can you share your code?
> 
> 
> 
> Also, why is the conversation marked as closed? I didn't see it until now. Did you resolve it yourself, or what? 😅

yep sorry, it has been resolved now. Was a mistake on my end:)
